AmpFinKit/Sources/AFBase/Extensions/Item/Track+Convert.swift

```diff
@@ -33,7 +33,7 @@ extension Track {
             artists: item.ArtistItems.map { ReducedArtist(id: $0.Id, name: $0.Name) },
             lufs: item.LUFS,
             index: Index(index: item.IndexNumber ?? fallbackIndex, disk: item.ParentIndexNumber ?? 1),
-            runtime: Double(item.RunTimeTicks / 10_000_000),
+            runtime: Double(item.RunTimeTicks != nil ? item.RunTimeTicks! / 10_000_000 : 0),
```

Comment:
This works but I used `(item.value ?? 0) / 10_000_000` in most places. I wanted to add SwiftLint at some point and I think it will complain about force unwraps, so this is a cleaner way of doing this.